src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -78,6 +140,26 @@ def _assess_clusters(self, all_clusters):
                 for value in cluster.spark_conf.values():
                     if "dbfs:/mnt" in value or "/dbfs/mnt" in value:
                         failures.append(f"using DBFS mount in configuration: {value}")
+
+                # Checking if Azure cluster config is present in spark config
+                for key in cluster.spark_conf.items():
+                    if azure_sp_conf_usage_check(str(key)):
```

Comment:
```suggestion
                for key in cluster.spark_conf.keys():
                    if azure_sp_conf_usage_check(key):
```
don't `str(...)` if you can get the real type

Reply:
sure.